tokenserver-auth/Cargo.toml

```diff
@@ -15,7 +15,7 @@ serde_json.workspace = true
 async-trait = "0.1.40"
 dyn-clone = "1.0.4"
 pyo3 = { version = "0.20", features = ["auto-initialize"] }
-reqwest = { workspace = true, features = ["json", "rustls-tls"] }
```

Comment:
Since rust features are additive, and we specify the rustls feature in the workspace, there is no need to specify it here